others/career-development.md

```diff
@@ -0,0 +1,73 @@
+# Career Development
+
+## CV
+
+### Best practice
+
+- Keep it short and concise, 1-2 pages maximum
+- Put your details at the top. Only these details are necessary:
+  - Name
+  - Email
+  - Phone number
+  - Your city (not your address since this is personal information! If they need it they'll ask for it later)
+- Use the following sections, and only populate them with relevant details starting from the most recent first
+  1. Education
+  2. Employment
+     - Use action verbs for each bullet point (see: <https://www.indeed.com/career-advice/resumes-cover-letters/action-verbs-to-make-your-resume-stand-out>)
+     - Show your accomplishments, with numbers if you can (e.g. Improved landing page time to interactive by 50%)
+  3. Skills
+  4. Personal projects
+  5. Personal interests (optional)
+
+### Resources
+
+Guide to writing a CV
+
+- <https://www.freecodecamp.org/news/writing-a-killer-software-engineering-resume-b11c91ef699d/>
```

Comment:
This is actually an American-style résumé, but since the article author has just graduated college, they don't have enough experience to make it much distinct from a UK CV.

Much of the advice is inappropriate for a CV. UK CVs are two pages, not one. Education shouldn't in general be at the top for anyone with actual experience (I think this is questionable for a US résumé also, but the author is a new college graduate...). Education coursework certainly shouldn't be on there if you have a reasonable quantity of real world experience (again this is probably also the case in the US market).

He does have good advice on showcasing accomplishments, but so much of the rest is misleading I would prefer not to use this link.

Reply:
Finding a good UK resource for this is surprisingly challenging. https://nationalcareers.service.gov.uk/get-a-job/cv-sections gives some general CV advice, but it's not at all tech-focused. http://www.energizerecruitment.co.uk/software-developers-guide-writing-cv-2017/ is decent but bare-bones (and like most such guides, assumes the standard degree -> work progression).

Reply:
Thanks, I've updated the CV resource links